src/main/java/org/uniovi/i3a/incimanager/rest/RESTController.java

```diff
@@ -41,23 +40,20 @@
 
 	@RequestMapping(value = "/sensor-feed", method = RequestMethod.POST, consumes = {
 			MediaType.APPLICATION_JSON_VALUE }, produces = MediaType.APPLICATION_JSON_VALUE)
-	public ResponseEntity<String> processSensorRequest( @RequestBody Map<String, Object> payload ) {
+	public ResponseEntity<Object> processSensorRequest( @RequestBody Map<String, Object> payload ) {
 		
 		// Authentication of the agent that made the request.
-		val authenticationResponse = agentsConnection.executeQuery( new AgentsQueryFormatter( payload ).query() );
+		HttpResponse<JsonNode> authenticationResponse = 
+				agentsConnection
+				.executeQuery( new AgentsQueryFormatter( payload ).query() );
 		
 		if(authenticationResponse.getStatus() != HttpStatus.OK.value()) {
-			log.warn( "[ERROR] UNAUTHORIZED ACCESS: trying to access as: "
-						+ payload.get( "login" ) + " "
-						+ payload.get( "password" ) + " "
-						+ payload.get( "kind" ));
-			
-			return new ResponseEntity<String>("{\"response\":\"UNAUTHORIZED ACCESS WILL BE REPORTED\"}", HttpStatus.UNAUTHORIZED );
```

Comment:
Seems like you just upload your changes without updating from master... Here the log is compulsory also.